src/components/Card.js

```diff
@@ -26,6 +28,7 @@ class Card extends React.PureComponent {
     text: PropTypes.string,
   };
 
+  /** @type {(url: string) => any} Typescript syntax */
```

Comment:
@vishalnarkhede here you can add types to the internal methods

Reply:
also good to keep in mind that classes will get killed in the hooks refactor ;)